src/network/index.ts

```diff
@@ -1 +1,73 @@
 /** @format */
+import { isEmpty } from "lodash";
+
+import * as Models from "./models";
+import { addHTTPHeaders, addQueryStringParams, buildApiUrl } from "./utils";
```

Comment:
multiline jslint for when single imports are over 1?

Reply:
Mentioned this above.